test/both/transport/connection-test.js

```diff
@@ -294,6 +294,30 @@ describe('Transport/Connection', function () {
       })
     })
 
+    it('should ignore WINDOW_UPDATE frame after END_STREAM', function (done) {
+      client.request({
+        path: '/hello'
+      }, function (err, stream) {
+        assert(!err)
+
+        stream.resume()
+
+        stream.once('close', function () {
+          client.on('frame', function (frame) {
+            assert(!(frame.type === 'RST' && frame.id === 1))
+          })
+
+          client._spdyState.framer.windowUpdateFrame({
+            id: stream.id,
+            delta: 1
+          }, function (err) {
+            assert(!err)
+            setTimeout(done, 10)
```

Comment:
Hi @diasdavid I wasn't very familiar with the architecture. The entire point of the PR is that the `frame` event won't fire any longer with a frame type of `RST` and `id` of 1. How should I test the absence of that emission?